### PR TITLE
Add materials subdomain

### PR DESCRIPTION
**Pull request by elichad**

### CODE REVIEW

File: templates/galaxy/config/global_host_filters.py.j2 [hunk 1]
```diff
@@ -85,7 +87,8 @@ DOMAIN_SECTIONS = {
     'metabolomics': ['metabolomics'],
     'ml': ['machine_learning'],
     'annotation': GENERAL_NGS_SECTIONS + ['annotation', 'apollo', 'ncbi_blast', 'assembly', 'variant_calling', 'fetch_sequences___alignments', 'ontology', 'emboss', 'evolution', 'genome_diversity', 'multiple_alignments', 'graph_display_data'],
-
+    'materials': ['materials_science'],
```
Comment (elichad) [inline templates/galaxy/config/global_host_filters.py.j2:90]: Not entirely sure what this is meant to be referencing - tool shed category?

Comment (bgruening) [reply templates/galaxy/config/global_host_filters.py.j2:90]: Tool categories ... Defined in the tool_conf.xml file

Comment (elichad) [reply templates/galaxy/config/global_host_filters.py.j2:90]: Aha, of course. Added.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
It should work tomorrow after redeployment, thanks @elichad!

**Pull request merged**